x-pack/plugins/security_solution/server/lib/app_features/mocks.ts

```diff
@@ -13,7 +13,7 @@ import { AppFeatures } from './app_features';
 import type { AppFeatureKeys, ExperimentalFeatures } from '../../../common';
 import { ALL_APP_FEATURE_KEYS, allowedExperimentalValues } from '../../../common';
 
-class AppFeaturesMock extends AppFeatures {
+export class AppFeaturesMock extends AppFeatures {
```

Comment:
I don't think you need to export this. you can continue to just use `AppFeatures` as a type

Reply:
FYI - https://github.com/elastic/kibana/pull/163602/commits/cd0c0847462c331f291064dc4a446e9d918de63d